src/System.Drawing.Common/src/System/Drawing/Icon.Windows.cs

```diff
@@ -14,6 +14,9 @@
 namespace System.Drawing
 {
     [Serializable]
+#if netcoreapp
```

Comment:
Icon has a Windows and a Unix file which don't share implementation. You need to add this attribute to the Icon.Unix file as well

Reply:
@safern There are some `#if`'s there and I do not know where/how tu put that attribute. Wherever I write it, I get some errors on my computer (I build corefx with `-BuildAllConfigurations` switch).